app/packs/src/models/Attachment.js

```diff
@@ -1,9 +1,9 @@
 import Element from 'src/models/Element';
 
 export default class Attachment extends Element {
-  filePreview(file) {
+  static filePreview(file) {
```

Comment:
Why is this still displayed as a change? development-5 should already have this change

Reply:
I think github uses ...

`git diff development-5...ci-image-update`